roles/common/tasks/k8s.yml

```diff
@@ -115,6 +115,21 @@
     gcloud config set project {{ project }}
     rm -f {{ keyfile }}
 
+# https://cloud.google.com/blog/products/containers-kubernetes/kubectl-auth-changes-in-gke
+# apt google-cloud-sdk-gke-gcloud-auth-plugin
+- name: Install gke-gcloud-auth-plugin if Debian
```

Comment:
I think this playbook _only_ support Debian anyway.

Reply:
Fixed